app/graphql/types/query_types.rb

```diff
@@ -34,5 +34,15 @@ def wells(plate_id: nil)
     def plates
       Plate.all
     end
+
+    # Ont
+
+    field :ont_libraries, [Types::Outputs::Ont::LibraryType], null: false do
+      description 'Find all Ont Libraries.'
+    end
+
+    def ont_libraries
+      Ont::Library.all
+    end
```

Comment:
I think the comment for the class on line 4 might need updating.  This must've been forgotten by me in a previous refactor

Reply:
Done